projects/isy-angular-widgets/src/lib/wizard/components/wizard/wizard.component.spec.ts

```diff
@@ -419,43 +408,39 @@ describe('Integration Tests: WizardComponent with Mock Parent', () => {
 
   it('should have a next button with some classes', () => {
     const nextButton = getNativeElementAsHTMLElement(nextButtonDeclaration);
+
     expect(nextButton.className).toContain(CLASS_ALIGN_CENTER);
     expect(nextButton.className).toContain(CLASS_JUSTIFY_CENTER);
     expect(nextButton.className).toContain(CLASS_MR_4);
   });
 
-  it('should have a functional next button', () => {
-    expectMovementToFirstStep();
-    expectMovementToNextStep(startIndex);
-  });
-
   it('should use the correct save button label', () => {
-    expectStepperMovedUntilEnd();
+    expectWizardMovedUntilEnd();
     const saveButton = getNativeElementAsHTMLElement(saveButtonDeclaration);
     expect(saveButton.innerHTML).toContain(wizard.labelSaveButton);
   });
 
   it('should have some classes on the save button', () => {
-    expectStepperMovedUntilEnd();
+    expectWizardMovedUntilEnd();
     const nextButton = getNativeElementAsHTMLElement(saveButtonDeclaration);
+
     expect(nextButton.className).toContain(CLASS_FLEX);
     expect(nextButton.className).toContain(CLASS_ALIGN_CENTER);
     expect(nextButton.className).toContain(CLASS_JUSTIFY_CENTER);
   });
 
   it('should have a visible save button on last step', () => {
-    expectStepperMovedUntilEnd();
-
+    expectWizardMovedUntilEnd();
     const saveButton = getNativeElementAsHTMLElement(saveButtonDeclaration);
     expect(saveButton).not.toBeNull();
   });
 
-  it('should have disabled save button on last step', () => {
+  it('should have a save button be disabled on last step', () => {
```

Comment:
last step not seen in test
why does this test work anyway?

Reply:
Done